config/DatabaseSetup.sql

```diff
@@ -5,13 +5,33 @@ CREATE DATABASE imp_db;
 
 USE imp_db;
 
+CREATE TABLE Users (
```

Comment:
We will want each User record to have a salt for hashing as well, typically a 16 character string. Its safe to store the salt with the user because all it can do is hash, it cant decode the password, and we generally want a different salt for each user to avoid rainbow table hacks.